simplq/src/components/common/LoginButton.jsx

```diff
@@ -26,6 +29,7 @@ const LoginButton = () => {
 
   const onLogoutCallback = () => {
     Auth.logOut();
+    history.push('/');
```

Comment:
Take the user back to homepage on logout. Otherwise, in the admin page, the unauthenticated user might not have permission, he will get Unauthorised error and will be confused.